uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointDocs.java

```diff
@@ -83,7 +83,7 @@ class ClientAdminEndpointDocs extends AdminClientCreator {
         fieldWithPath("jwks").constrained("Optional only, if jwks_uri is used. Required otherwise.").type(STRING).description("A valid JSON string according JSON Web Key Set standard, see [RFC 7517](https://www.rfc-editor.org/rfc/rfc7517), e.g. content of /token_keys endpoint from UAA"),
         fieldWithPath("jwks_uri").constrained("Optional only, if jwks is used. Required otherwise.").type(STRING).description("A valid URI to token keys endpoint. Must be compliant to jwks_uri from [OpenID Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html)."),
         fieldWithPath("kid").constrained("Optional only, if a single JWK should be deleted, else ignored.").type(STRING).description("If change mode is set to `"+ClientJwtChangeRequest.ChangeMode.DELETE+"`, the `id of the key` that will be deleted. The kid parameter is only possible if jwks configuration is used."),
-        fieldWithPath("changeMode").optional(ClientJwtChangeRequest.ChangeMode.ADD).type(STRING).description("If change mode is set to `"+ClientJwtChangeRequest.ChangeMode.ADD+"`, the new `JWKS` or `JWKS_URI` will be added to the existing configuration and if the change mode is set to `"+ClientJwtChangeRequest.ChangeMode.DELETE+"`, the old JWK or JWKS_URI will be deleted.")
+        fieldWithPath("changeMode").optional(ClientJwtChangeRequest.ChangeMode.ADD).type(STRING).description("If change mode is set to `"+ClientJwtChangeRequest.ChangeMode.ADD+"`, the new `JWKS` will be added to the existing configuration and if the change mode is set to `"+ClientJwtChangeRequest.ChangeMode.DELETE+"`, the old JWK will be deleted. The option `"+ClientJwtChangeRequest.ChangeMode.UPDATE+"` changes the complete trust setting and allow to switch between `JWKS` and `JWKS_URI`.")
```

Comment:
Minor: 
May be this could paraphrased as follows:

The option 'UPDATE' enables changes to the complete trust setting and allows switching between JWKS and JWKS_URI.

Reply:
done